zulip_bots/zulip_bots/bots/mention/test_mention.py

```diff
@@ -43,3 +44,8 @@ def test_get_mentions(self) -> None:
             bot_response = bot_test_instance.get_mentions('TEST')[0]
             self.assertEqual(bot_response['title'], 'TEST')
             self.assertEqual(bot_response['original_url'], 'TEST')
+
+    def test_exception_when_api_key_is_invalid(self) -> None:
+        with self.mock_config_info({'access_token': 'TEST'}):
+            with self.assertRaises(StubBotHandler.BotQuitException):
```

Comment:
See my other comment.  Why are supplying an expected reply here if we expect this to raise an exception, and why aren't we testing a more specific thing like just initializing?

Reply:
Did this to to use `context manager` to cover line 33 in `baremetrics.py`.  Just initializing was not covering that line.
[Code Coverage Report](https://codecov.io/gh/zulip/python-zulip-api/src/master/zulip_bots/zulip_bots/bots/baremetrics/baremetrics.py)

Reply:
You are right here as well. Will change it so that we test just the initialization, and notify. Thanks for your review!